public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans|Source+Sans+Pro&display=swap" rel="stylesheet">
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://avatars3.githubusercontent.com/u/62974140?s=200&v=4" />
```

Comment:
Instead of grabbing it from our GitHub organization, you should be able to give the filepath to the icon the `src/assets/` directory which is where I uploaded it from; that way it'd be less tied into our GitHub account.

Reply:
Alright, will do.